docs/source/use_cases/analyze_reads/calculate_coverage.rst

```diff
@@ -45,6 +48,13 @@ write access, you may use it.  If not, you can do the following to create one:
 The ``id`` that is generated in the response output is the output dataset id you should use when running
 the pipeline.
 
+Download ALPN
+-------------
+
+Running this pipeline locally requires that you have the ALPN jar that matches your JRE version
+on your computer.  See the `ALPN documentation <http://www.eclipse.org/jetty/documentation/9.2.10.v20150310/alpn-chapter.html>`_ for a table of which ALPN jar to use.
+
+You will not need to provide the ALPN flag or jar if you run the pipeline on Google cloud instead of locally.
```

Comment:
@Careyjmac  Actually you don't need to set it, since by default it is set to `true` via [line 147 in CalculateCoverage.java](https://github.com/googlegenomics/dataflow-java/blob/93622c89b058636e14b1e13b55980cd1eb5e9780/src/main/java/com/google/cloud/genomics/dataflow/pipelines/CalculateCoverage.java#L147):

```Java
@Default.Boolean(true)
Boolean getUseAlpn();
```	

For instance, I did a quick test and updated `main` in `CalculateCoverage.java` with the following:

```Java
    LOG.info("use_alpn: " + options.getUseAlpn().toString());
    System.out.println("The current ALPN setting is => " + options.getUseAlpn().toString());
```

When I run it, it knows the result in both cases (i.e. being set or not):

```Bash
$ mvn package
$ java -cp target/google-genomics-dataflow*runnable.jar \
  com.google.cloud.genomics.dataflow.pipelines.CalculateCoverage \
  --project=<removed> \
  --secretsFile=client_secrets.json \
  --references=1:552960:553984 \
  --bucketWidth=1024 \
  --inputDatasetId=10473108253681171589 \
  --useAlpn=false
Jun 11, 2015 5:13:01 PM com.google.cloud.genomics.dataflow.pipelines.CalculateCoverage main
INFO: use_alpn: false
The current ALPN setting is => false
$
$ java -cp target/google-genomics-dataflow*runnable.jar \
  com.google.cloud.genomics.dataflow.pipelines.CalculateCoverage \
  --project=<removed> \
  --secretsFile=client_secrets.json \
  --references=1:552960:553984 \
  --bucketWidth=1024 \
  --inputDatasetId=10473108253681171589 
Jun 11, 2015 5:13:40 PM com.google.cloud.genomics.dataflow.pipelines.CalculateCoverage main
INFO: use_alpn: true
The current ALPN setting is => true
$
```

Could you also share Sam's document?  My email is pgrosu@gmail.com

Thanks,
Paul